docker-compose example/docker-compose.example.yml

```diff
@@ -104,11 +116,26 @@ services:
     container_name: photoview-mariadb
     restart: unless-stopped
     stop_grace_period: 5s
+    user: "999:999"
     ## Optimized MariaDB startup command for better performance and compatibility
     command: mariadbd --innodb-buffer-pool-size=512M --transaction-isolation=READ-COMMITTED --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci --max-connections=512 --innodb-rollback-on-timeout=OFF --innodb-lock-wait-timeout=120
-    security_opt: ## see https://github.com/MariaDB/mariadb-docker/issues/434#issuecomment-1136151239
-      - seccomp:unconfined
-      - apparmor:unconfined
+
```

Comment:
Please remove empty lines inside the same service definition: it is a styling issue, as in this way we could visually identify where the definition of a previous service ends and a new one starts: we have pretty long definitions (including comments), so I'd like to keep the structure as clear as possible)

Reply:
done in https://github.com/photoview/photoview/pull/992/commits/0719a1e3b663fc03aae700bd43fb3b8c0eccbe5f